### PR TITLE
login screen

### PR DESCRIPTION
**Pull request by mosayed01**

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing_hug"
+        android:text="@string/welcome_back"
+        android:textColor="@color/text_secondary_60"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView2" />
+
+    <EditText
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/activity_login.xml]: create style to EditText and use in two EditText

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (AmeerAmjed) [inline app/src/main/res/layout/activity_login.xml]: image
image

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -1,214 +1,214 @@
...
+                android:layout_gravity="end"
+                android:text="@string/skip"
+                android:textColor="@color/text_tertiary_38" />
+
+        </com.google.android.material.appbar.AppBarLayout>
+
+
+        <ImageView
+            android:id="@+id/imageView2"
+            android:layout_width="131dp"
```
Comment (TarekIdrees) [inline app/src/main/res/layout/activity_login.xml]: width and height of photo may be extracted to file strings

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -1,214 +1,214 @@
...
-        android:layout_width="@dimen/medium_icon_size"
-        android:layout_height="@dimen/medium_icon_size"
-        android:layout_marginEnd="@dimen/spacing_x_large"
-        android:background="@color/background"
-        android:contentDescription="@string/sign_in_with_google"
-        android:src="@drawable/icon_google"
-        app:layout_constraintEnd_toStartOf="@id/twitter_login"
-        app:layout_constraintTop_toTopOf="@id/twitter_login" />
+            android:layout_marginTop="@dimen/spacing_small"
+            android:layout_marginEnd="16dp"
```
Comment (TarekIdrees) [inline app/src/main/res/layout/activity_login.xml]: margin end hardcoded

File: app/src/main/res/layout/activity_login.xml [hunk 5]
```diff
@@ -1,214 +1,214 @@
...
+            android:text="@string/forget_your_password"
+            android:textColor="@color/text_secondary_60"
+            android:textSize="@dimen/font_size_caption"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/textView12" />
 
+        <TextView
+            android:id="@+id/textView10"
```
Comment (TarekIdrees) [inline app/src/main/res/layout/activity_login.xml]: use meaningful name with id

**Review by TarekIdrees — CHANGES_REQUESTED** [review]
please take care of the requested changes

**Review by TarekIdrees — APPROVED** [review]

**Review by AmeerAmjed — APPROVED** [review]

**Pull request merged**